Cargo.toml

```diff
@@ -52,6 +52,13 @@ unused_unsafe = "warn"
 missing_unsafe_on_extern = "warn"
 unsafe_attr_outside_unsafe = "warn"
 
+# == Correctness == #
+ambiguous_negative_literals  = "warn"
+arithmetic_overflow = "warn"
+future_incompatible = "warn"
```

Comment:
question: Is it really a correctness lint? I understand this lint is more about warning about code that will stop compiling in the future. It’s also my understanding that all these lints are warn-by-default (or deny-by-default), so we don’t have to enable it again here. Did I miss something?

Reply:
> It’s also my understanding that all these lints are warn-by-default (or deny-by-default), so we don’t have to enable it again here. Did I miss something?

Ooh, no...  I thought they're disabled by default.